app/views/users/_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= form_for @user do |f| %>
```

Comment:
Nguyên văn: When constructing a form using form_for(@user), Rails uses POST if @user.new_record? is true and PATCH if it is false.
=> Nếu nó trả về true thì là Create, còn false thì là Update..